tests-e2e/core/applicationsnapshotenvironmentbinding_test.go

```diff
@@ -362,6 +362,10 @@ var _ = Describe("ApplicationSnapshotEnvironmentBinding Reconciler E2E tests", f
 
 		It("should create a GitOpsDeployment that references cluster credentials specified in Environment", func() {
 
+			if fixture.IsRunningAgainstKCP() {
+				Skip("Skipping this test until we support running gitops operator with KCP")
```

Comment:
I think this error message should be updated to indicate it is a race condition when running on KCP, rather than related to gitops operator, unless I am misunderstanding the issue.

Can you also open a quick JIRA task with a title like 'chore: fix race condition in applicationsnapshotenvrionmentbinding_test.go', pointing to this comment?

Reply:
Aha absolutely I'll do both, silly me :)